### PR TITLE
Put Switch Around Redirect of Expired Commercial Content

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This is the first step in removing the frontend check for commercially expired content.
The check is now being done in the backend and arriving through Capi.

I'm putting this behind a switch rather than removing it immediately just in case there are problems.
We should know in a month if it's safe to remove altogether.

### CODE REVIEW

File: common/app/model/meta.scala [hunk 1]
```diff
@@ -248,13 +254,13 @@ trait Elements {
     showcasePicture.getOrElse(false) || showcaseEmbed.getOrElse(false)
   }
 
-  def mainVideo: Option[VideoElement] = videos.find(_.isMain).headOption
```
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: This was suggested by Intellij.

File: common/app/model/meta.scala [hunk 2]
```diff
@@ -248,13 +254,13 @@ trait Elements {
...
+  def mainVideo: Option[VideoElement] = videos.find(_.isMain)
   lazy val hasMainVideo: Boolean = mainVideo.flatMap(_.videoAssets.headOption).isDefined
 
-  def mainAudio: Option[AudioElement] = audios.find(_.isMain).headOption
```
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: This was suggested by Intellij.

File: common/app/model/meta.scala [hunk 3]
```diff
@@ -248,13 +254,13 @@ trait Elements {
...
+  def mainAudio: Option[AudioElement] = audios.find(_.isMain)
   lazy val hasMainAudio: Boolean = mainAudio.flatMap(_.audioAssets.headOption).isDefined
 
-  def mainEmbed: Option[EmbedElement] = embeds.find(_.isMain).headOption
```
Comment (kelvin-chappell) [inline common/app/model/meta.scala]: This was suggested by Intellij.

File: common/app/model/meta.scala (hunk 1 above)
Comment (jennysivapalan) [inline common/app/model/meta.scala]: :+1: Intellij

**Comment by jennysivapalan** [comment]
:+1:

**Pull request merged**